Automattic/wp-calypso
# Update Masterbar Reader icon

Related to https://github.com/Automattic/dotcom-forge/issues/8344

## Proposed Changes

* This PR updates the Reader icon in the Masterbar in Calypso

Before | After
--|--
<img width="447" alt="Screenshot 2024-07-19 at 4 28 07 PM" src="https://github.com/user-attachments/assets/2826ec3f-7758-47a9-bba3-2a4ee948694a">  | <img width="443" alt="Screenshot 2024-07-19 at 4 26 13 PM" src="https://github.com/user-attachments/assets/6b558c25-8c4a-4df1-8df5-7c72d4fabe2f">


Before | After
--|--
<img width="1036" alt="Screenshot 2024-07-19 at 4 27 31 PM" src="https://github.com/user-attachments/assets/add5fd9f-6266-4a8e-9fe0-c75c53a89579"> | <img width="1034" alt="Screenshot 2024-07-19 at 4 26 39 PM" src="https://github.com/user-attachments/assets/a5d5a7e7-0fa0-462c-ab47-42b9115103c3">


## Why are these changes being made?


* To update branding.

## Testing Instructions



* Load Calypso live and go to /sites
* View the Masterbar anywhere in Calypso, it should have the new icon.
* Test it in mobile.
* Try clicking on it.

## Pre-merge Checklist
